legal-api/report-templates/incorporationApplication.html

```diff
@@ -51,32 +52,37 @@
             <div class="separator"></div>
             [[incorporation-application/completingParty.html]]
          </div>
-         <div class="section-header">
-            <div class="section-header-content">NOTICE OF ARTICLES</div>
-         </div>
-         <div class="container">
-            <div class="section-data benefit-company-statement">
-               <div class="section-title">Benefit Company Statement</div>
-               <div class="section-data">
-                  <p>This company is a benefit company and, as such, has purposes that include conducting its
-                     business in a responsible and sustainable manner and promoting one or more public benefits.
-                  </p>
-                  <p class="section-sub-title">Name of Company:
-                     {% if not incorporationApplication.nameRequest%}
-                     {{business.identifier}} B.C. LTD.
-                     {% else %}
-                     {{business.legalName}}
-                     {% endif %}
-                  </p>
-               </div>
+      </div>
+      {% endif %}
+      <div class="section-header">
+         <div class="section-header-content">NOTICE OF ARTICLES</div>
+      </div>
+      <div class="container">
+         <div class="section-data benefit-company-statement">
+            <div class="section-title">Benefit Company Statement</div>
+            <div class="section-data">
+               <p>This company is a benefit company and, as such, has purposes that include conducting its
+                  business in a responsible and sustainable manner and promoting one or more public benefits.
+               </p>
+               <p><span class="section-sub-title">Name of Company:</span>
+                  {% if not incorporationApplication.nameRequest%}
+                  {% if header.reportType == 'noa'%}
```

Comment:
Can nested if's be nested for easier readability?

Reply:
I have seen this kind of style through out old files as well. So I followed the same approach :)